test/helper.js

```diff
@@ -0,0 +1,37 @@
+var helper = require('../lib/helper');
+var expect = require('chai').expect;
+
+describe('helper', function() {
+  describe('parseLoopBackVersion()', function() {
+    it('should return valid loopback version', function() {
```

Comment:
Please improve test names to only mention what's the outcome ("return valid loopback version"), but also what are the conditions when this outcome is expected. Most importantly, the test name should make it clear whether the input was a version, a version range, or a git url.